src/pages/CustomerInvoicesPage.js

```diff
@@ -61,30 +49,41 @@ export const CustomerInvoicesPage = ({
     dataState,
     sortBy,
     isAscending,
-    columns,
     modalKey,
     hasSelection,
     keyExtractor,
     searchTerm,
+    columns,
+    PageActions,
   } = state;
 
-  // Refresh data on navigating back to this page.
-  useNavigationFocusRefresh(dispatch, navigation);
+  // Listen to changes from sync and navigation events re-focusing this screen,
+  // such that any side effects that occur trigger a reconcilitation of data.
+  const refreshCallback = () => dispatch(PageActions.refreshData());
+  useNavigationFocus(refreshCallback, navigation);
+  useSyncListener(refreshCallback, ['Transaction']);
```

Comment:
Do you even need this? You shouldn't be receiving CI's through sync, anyway?

Reply:
🤷‍♂